stackslib/src/config/mod.rs

```diff
@@ -97,7 +97,8 @@ const DEFAULT_TENURE_COST_LIMIT_PER_BLOCK_PERCENTAGE: u8 = 25;
 const DEFAULT_TENURE_EXTEND_POLL_SECS: u64 = 1;
 
 // This should be greater than the signers' timeout. This is used for issuing fallback tenure extends
-const DEFAULT_TENURE_TIMEOUT_SECS: u64 = 420;
+const DEFAULT_TENURE_TIMEOUT_SECS: u64 = 180;
+const DEFAULT_TENURE_EXTEND_COST_THRESHOLD: u64 = 50;
```

Comment:
Can you add a comment here (with a note that this is a percentage)?

Reply:
I added comments for all of them in 6fccd18d9dfeccf1035d1b48a66062e4b57f3132.